system/datatypes/mailinglist.php

```diff
@@ -1083,6 +1151,25 @@ protected function mailingSendTestAction(HyphaRequest $request) {
 		return 'reload';
 	}
 
+	protected function mailingProgressAction(HyphaRequest $request) {
+		if (!$this->hasSender()) {
```

Comment:
Is this check really relevant here? Seems like a copy-paste that is not really needed here?

Reply:
A copy paste  indeed. I kept it as it seemed to be used in other functions. It made sense to me that a mailing without a sender would fail to return progress. 
Should I remove it?

Reply:
I would remove it for simplicity.

Reply:
Done :)